crates/oxc_transformer/src/typescript/enum.rs

```diff
@@ -82,12 +82,13 @@ impl<'a> TypeScriptEnum<'a> {
             func_scope_id,
             AstNodeId::DUMMY,
         );
+        ctx.scopes_mut().add_binding(func_scope_id, enum_name.to_compact_str(), param_symbol_id);
```

Comment:
`TraverseCtx` should provide a method to do the same thing

Reply:
There are tons of methods on `ScopeTree`. I didn't attempt to add them all to `TraverseCtx` - only added methods to `TraverseCtx` where they didn't already exist on `ScopeTree` / `SymbolTable`.

Ideally I think we should go in the other direction - move as many methods as possible from `TraverseCtx` into `Semantic` (but there are some kinks, because `Traverse` currently doesn't have access to `AstNodes`).

Reply:
> There are tons of methods on `ScopeTree`. I didn't attempt to add them all to `TraverseCtx` - only added methods to `TraverseCtx` where they didn't already exist on `ScopeTree` / `SymbolTable`.
> 
I see
> Ideally I think we should go in the other direction - move as many methods as possible from `TraverseCtx` into `Semantic`. But there are some kinks, because `Traverse` currently doesn't have access to `AstNodes` - have been trying to figure out how to deal with that.

Yes